user-management.yaml

```
@@ -64,6 +64,12 @@ paths:
         required: false
         schema:
           type: "string"
+      - name: "only_active"
+        in: "query"
+        description: "A boolean that determines if only active user or only inactive users should be fetched"
```

```suggestion
        description: "A boolean that determines if only active users or both active and inactive users should be fetched"
```
Correct me if I am wrong, but this is how I would understand the flag.

---

I will correct you since you are wrong, if the parameter is false, we return ONLY the deleted users. We should rename